changelog/729.doc.rst

```diff
@@ -0,0 +1 @@
+Added docstrings to the file `PlasmaPy/plasmapy/utils/decorators/converter.py`.
```

Comment:
Change to...

Add docstrings to decorator :func:`plasmapy.utils.decorators.converter.angular_freq_to_hz`